aggregator/src/test/java/org/jenkinsci/plugins/workflow/steps/MilestoneStepTest.java

```diff
@@ -0,0 +1,106 @@
+package org.jenkinsci.plugins.workflow.steps;
+
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.jenkinsci.plugins.workflow.test.steps.SemaphoreStep;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runners.model.Statement;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+import hudson.model.Result;
+
+public class MilestoneStepTest {
+
+    @Rule
+    public RestartableJenkinsRule story = new RestartableJenkinsRule();
```

Comment:
Recommend adding `BuildWatcher`—makes it much easier to develop & debug this kind of test.